vite.config.ts

```diff
@@ -20,6 +20,7 @@ export default defineConfig({
     preprocessorOptions: {
       scss: {
         additionalData: `
+          @import "./src/styles/_reset.scss";
```

Comment:
If we use normalize, it will be better to rename the file

Reply:
Done